scripts/build-color-variants.js

```diff
@@ -480,11 +480,13 @@ function buildColorVariants(variables, config) {
       const colorValue = variables[color];
       return result += stripIndent(`
         .shadow-${color}-on-hover:hover,
-        .shadow-${color}-on-active.is-active {
+        .shadow-${color}-on-active.is-active,
+        .shadow-${color}-on-active.is-active:hover {
```

Comment:
Discussed in chat with @davidtheclark: this looks like the better path to go down.